vendor/comit_i/build.rs

```diff
@@ -0,0 +1,16 @@
+use std::fs::File;
+
+fn main() {
+    let version = include_str!("./version").trim();
```

Comment:
It might be useful to do some logging here like
- which version of comit-i we are downloading
- where the tmp file is located

Reference: https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script